### PR TITLE
Add a tick box to the proposal form indicating 1st-time speaker

### PR DESCRIPTION
**Pull request by alice6373**
## Add a tick box to the proposal form indicating 1st-time speaker
- [ ] **Bugfix**
- [ ] **New feature**
- [ ] **Refactoring**
- [ ] **Breaking change** (any change that would cause existing functionality to not work as expected)
- [ ] **Documentation Update**
- [x] **Enhancement**

## Description
Support program team's plan of young talent. 
Add a tick box in talk proposal & tutorial proposal form for users indicating whether she/he is the first time speaker.

## Steps to Test This Pull Request
1. Go to 'MY PYCON'
2. Click on 'proposals' tab
4. Click on '+New Talk Proposal' or '+New Tutorial Proposal''
3. Scroll down
4. See tick box of *First time speaker*

## Expected behavior
After the proposal form is submitted, the newly added data in the talk proposal and toturial proposal tables will contain information about whether this user is the first time speaker.

## More Information
**Screenshots**
- Frontend

image

- Django Administration
image
image

### CODE REVIEW

File: src/proposals/forms/mixins.py [hunk 1]
```diff
@@ -227,4 +231,8 @@ class Meta:
                 "(coronavirus) is still raging this August?"
 
             ),
+            'first_time_speaker': _(
```
Comment (mattwang44) [inline src/proposals/forms/mixins.py:234]: It's my fault that I didn't mention that we do not this tickbox for the tutorials. Please delete the changes related to the tutorial model.

File: src/core/models.py [hunk 2]
```diff
@@ -196,6 +196,16 @@ class EventInfo(models.Model):
...
+    FIRST_SPEAKER_CHOICES = (
+        (True, _('Yes')),
+        (False, _('No'))
+    )
+
+    first_time_speaker = models.BooleanField(
+        verbose_name=_('first time speaker'),
+        default=False,
+        choices=FIRST_SPEAKER_CHOICES)
+
```
Comment (mattwang44) [inline src/core/models.py:208]: `EventInfo` is a general model inherited by many other models. I think these lines should be moved to a model in `proposals/models.py`.

**Comment by mattwang44** [comment]
CI failed. The root cause should be your branch is not up-to-date.

**Comment by mattwang44** [comment]
[Add a tick box to the proposal form indicating 1st-time speaker](https://trello.com/c/6Nzwn0fP/20-add-a-tick-box-to-the-proposal-form-indicating-1st-time-speaker)

File: src/proposals/forms/mixins.py [hunk 3]
```diff
@@ -130,6 +130,10 @@ class Meta:
...
+                "Is it the first time to be the speaker of pycon?"
```
Comment (mattwang44) [inline src/proposals/forms/mixins.py:134]: Note that we only care about whether the submitter has ever given any talk in _PyCon TW_. e.g. If one was a speaker in, say, PyCon JP before, he/she still can be the a _first-time speaker of PyCon TW_. 

```suggestion
                "Is it your first time to be the speaker of PyCon TW?"
```

File: src/proposals/forms/mixins.py (hunk 1 above)
Comment (alice6373) [reply src/proposals/forms/mixins.py:234]: Ok. I have deleted the changes related to the tutorial model.

File: src/core/models.py (hunk 2 above)
Comment (alice6373) [reply src/core/models.py]: Thanks for your reminder. I revise it.

File: src/proposals/admin.py [hunk 4]
```diff
@@ -62,6 +60,29 @@ class ProposalAdmin(admin.ModelAdmin):
 @admin.register(TalkProposal)
 class TalkProposalAdmin(ExportMixin, ProposalAdmin):
     resource_class = TalkProposalResource
+    fields = [
```
Comment (mattwang44) [inline src/proposals/admin.py:63]: why do we need this list? which field(s) do you want to exclude?

**Review by mattwang44 — APPROVED** [review]
LGTM

**Pull request merged**